src/modules/projects/components/EsgFundingReport.tsx

```diff
@@ -0,0 +1,149 @@
+import { Paper, Typography } from '@mui/material';
+import { isNil } from 'lodash-es';
+import { useMemo } from 'react';
+
+import Loading from '@/components/elements/Loading';
+import GenericTable from '@/components/elements/table/GenericTable';
+import PageTitle from '@/components/layout/PageTitle';
+import PrintViewButton from '@/components/layout/PrintViewButton';
+import NotFound from '@/components/pages/NotFound';
+import useIsPrintView from '@/hooks/useIsPrintView';
+import usePrintTrigger from '@/hooks/usePrintTrigger';
+import useSafeParams from '@/hooks/useSafeParams';
+import ApolloErrorAlert from '@/modules/errors/components/ApolloErrorAlert';
+import { formatCurrency, parseAndFormatDate } from '@/modules/hmis/hmisUtil';
+import {
+  EsgFundingServiceFieldsFragment,
+  useGetEsgFundingReportQuery,
+  useGetReferralPostingQuery,
+} from '@/types/gqlTypes';
+
+const EsgFundingReport: React.FC = () => {
+  const { referralPostingId } = useSafeParams<{ referralPostingId: string }>();
+  const { data, loading, error } = useGetReferralPostingQuery({
+    variables: { id: referralPostingId as any as string },
+    fetchPolicy: 'network-only',
```

Comment:
Why use network only? We definitely have it in the cache already when navigating to the ESG funding report, so I don't think we want to refetch it.

Reply:
Sounds good, I'll switch it